src/test/java/org/fiware/vc/it/HappypetsToPacketDeliveryStandardScenario.java

```diff
@@ -41,21 +42,47 @@ public void setupApplication() throws Exception {
 						PacketDeliveryEnvironment.PACKET_DELIVERY_DID,
 						PacketDeliveryEnvironment.PACKET_DELIVERY_VERIFIER_ADDRESS,
 						"/token",
-						PacketDeliveryEnvironment.PACKET_DELIVERY_ORION_ADDRESS));
+						PacketDeliveryEnvironment.PACKET_DELIVERY_ORION_ADDRESS,
+						PacketDeliveryEnvironment.PACKET_DELIVERY_SERVICE_ID));
+	}
+
+	@Given("HappyPets is a trusted issuer in PacketDelivery.")
+	public void registerHappyPetsAtPacketDelivery() throws Exception {
+		// In order to be accepted by PacketDelivery, HappyPets needs to be registerd at the Trusted Issuers List of PacketDelivery.
+		// In a real world use-case, that might happen through a marketplace, where HappyPets purchased the access for its users.
+		userEnvironment.getApplication().registerTrustedIssuer(HappyPetsEnvironment.HAPPYPETS_DID);
+	}
+
+	@Given("The packet-delivery portal is registered at the config-service.")
+	public void registerThePortal() throws Exception {
+		// the portal has to be registered as a client at the verifier's config service
+		userEnvironment.getApplication().registerCredentialsConfig();
+		// we need to wait for the config service to get the credentials
```

Comment:
Can we increase the update rate of the config service? >30s is quite a long time for the test.

Reply:
CCS doesn't have any delay building but the verifier does. @beknazaresenbek is currently working on the credential-config cache so I would delay making it configurable till his PR is done